Common/src/CConfig.cpp

```diff
@@ -5342,6 +5354,33 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
     case TURB_FAMILY::KW:
       nTurbVar = 2; break;
   }
+  /*--- Check whether the number of entries of the MARKER_INLET_TURBULENT equals the number of turbulent properties
+       used for the respective turbulent model. nTurb_Properties must be equal to 1 or 2 depending on whether SA or
+       SST model are used.--- */
+  if (Marker_Inlet_Turb != nullptr && Kind_Turb_Model == TURB_MODEL::SST && nTurb_Properties != 2)
+    SU2_MPI::Error(
+        "The use of MARKER_INLET_TURBULENT requires the number of entries when SST Model is used \n"
+        "to be equal to 2 : Turbulent intensity and ratio turbulent to laminar viscosity",
+        CURRENT_FUNCTION);
+
+  /* if Turbulent Inlet is not provided, MARKER_INLET_TURBULENT will be filled with the turbulent properties
+   provided in the .cfg file when SST model is used in order to recover default implementation in SU2*/
+
+  if (Marker_Inlet_Turb == nullptr && Kind_Turb_Model == TURB_MODEL::SST) {
+    string* Inlet_Options[5] = {Marker_Inlet, Marker_Riemann, Marker_Giles, Marker_Supersonic_Inlet,
+                                Marker_ActDiskInlet};
+    for (unsigned short i = 0; i < 5; i++) {
+      if (Inlet_Options[i] != nullptr) Marker_Inlet_Turb = Inlet_Options[i];
+    }
+    nMarker_Inlet_Turb =
+        nMarker_Inlet + nMarker_Riemann + nMarker_Giles + nMarker_Supersonic_Inlet + nMarker_ActDiskInlet;
+    Inlet_TurbVal = new su2double*[nMarker_Inlet_Turb];
+    for (unsigned short iMarker = 0; iMarker < nMarker_Inlet_Turb; iMarker++) {
+      Inlet_TurbVal[iMarker] = new su2double[2]();
+      Inlet_TurbVal[iMarker][0] = TurbulenceIntensity_FreeStream;
+      Inlet_TurbVal[iMarker][1] = Turb2LamViscRatio_FreeStream;
```

Comment:
Instead of doing this you could just return the default values in `GetInlet_TurbVal` if the name is not found. As you have it now there would be strange failures if someone doesn't specify all the markers.

Reply:
Hi Pedro, Thank you for the comment. I have done what you suggest. Please let me know if you have any advice/suggestions about how this can be improved. Thank you so much in advance.

Reply:
Hi Pedro, thank you so much for your fix in returning the freestream values in the GetInlet_TurbVal!!